apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx

```diff
@@ -135,6 +143,49 @@ const AssetScreen = ({ route }: NavigationProps) => {
     }
   }, [currency, navigation]);
 
+  // Dynamic Content Part -------------------
+  const {
+    getAssetCardByIdOrTicker,
+    logDismissCard,
+    logClickCard,
+    logImpressionCard,
+  } = useDynamicContent();
+  const dynamicContentCard = getAssetCardByIdOrTicker(currency);
+
+  const onClickLink = useCallback(
+    (card: ContentCard) => {
+      track("contentcard_clicked", {
+        screen: card.location,
+        link: card.link,
+      });
+      // Notify Braze that the card has been clicked by the user
+      logClickCard(card.id);
+      Linking.openURL(card.link);
+    },
+    [logClickCard],
+  );
+
+  const onPressDismiss = useCallback(
+    (dismissAction: () => void, card: ContentCard) => {
+      track("contentcard_dismissed", {
+        screen: card.location,
+      });
+      // Braze dismiss  logDismissCard(card.id);
+      dismissAction();
```

Comment:
We need to remove this line as we're keeping the current behavior

Reply:
`dismissAction` this one is the function that you pass as param

`logDismissCard(card.id);` it's this that i commented :)